libs/langchain/langchain/llms/octoai_endpoint.py

```diff
@@ -132,6 +132,11 @@ def _call(
                 # Send the request using the OctoAI client
                 output = octoai_client.infer(self.endpoint_url, parameter_payload)
                 text = output.get("choices")[0].get("message").get("content")
+                if (
```

Comment:
we want to append the tokens used to the completion string? that seems weird... why?